package/MDAnalysis/analysis/align.py

```diff
@@ -1159,15 +1177,29 @@ def fasta2select(fastafilename, is_aligned=False,
     :func:`sequence_alignment`, which does not require external
     programs.
 
+    
+    Raises
+    ------
+    ImportError
+      If optional dependency biopython is not available.
+
 
     .. _ClustalW: http://www.clustal.org/
     .. _STAMP: http://www.compbio.dundee.ac.uk/manuals/stamp.4.2/
 
     .. versionchanged:: 1.0.0
        Passing `alnfilename` or `treefilename` as `None` will create a file in
        the current working directory.
+    .. versionchanged:: 2.7.0
+       Biopython is now an optional dependency which this method requires.
 
     """
+    if not HAS_BIOPYTHON:
+        errmsg = ("The `fasta2select` method requires an installation "
+                  "of biopython. Please install biopython to use this method: "
```

Comment:
```suggestion
                  "of Biopython. Please install Biopython to use this method: "
```

Maybe this message could be written once as module variable? `missing_biopython_errmsg` or something on those lines?

Reply:
missed that comment - could do but we're using slightly different method names. I don't have any strong feelings here so I'm happy to change things up if you think it's worth it.

Reply:
Oh, right. I missed the different method name. Never mind.

Reply:
Might be a good use of string templates, but I don't think it's worth it.